src/main/java/io/vertx/junit5/VertxTestContext.java

```diff
@@ -221,6 +222,48 @@ public synchronized Checkpoint checkpoint(int requiredNumberOfPasses) {
 
   // ........................................................................................... //
 
+  /**
+   * This method allows you to check if a future is completed. It internally creates a checkpoint. You can use it in a future chain
```

Comment:
I'd format 1 line per sentence, then don't forget to end the line with a dot.

```suggestion
   * This method allows you to check if a future is completed. 
   * It internally creates a checkpoint.
   * You can use it in a chain of `Future` method calls.
```